x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/i18n_texts.ts

```diff
@@ -9,9 +9,13 @@ import { i18n } from '@kbn/i18n';
 
 export const i18nTexts = {
   editPolicy: {
-    shrinkLabel: i18n.translate('xpack.indexLifecycleMgmt.shrink.indexFieldLabel', {
+    shrinkLabel: i18n.translate('xpack.indexLifecycleMgmt.shrink.enableShrinkLabel', {
       defaultMessage: 'Shrink index',
     }),
+    shrinkCountOrSizeTooltip: i18n.translate('xpack.indexLifecycleMgmt.shrink.countOrSizeTooltip', {
+      defaultMessage:
+        'Set a number of shards or the maximum primary shard size for the target index.',
```

Comment:
I think we can shorten this a bit.
```suggestion
        'Set the number of primary shards or maximum shard size for the index.',
```

I was a little confused by the display here. The text anchor changes, but the tool tip stays the same. Can we change the tool tip based on the anchor text?

<img width="551" alt="Screen Shot 2021-09-10 at 2 19 21 PM" src="https://user-images.githubusercontent.com/40268737/132899944-56fb48f7-cab4-4e75-b405-9b584d7abfa0.PNG">